### ISSUE TITLE
Chat "login" strangely only works sometimes, and only for "big" collabs

### ISSUE THREAD

**Issue by LittleVulpix** [HolodexNet/Holodex#458]
Firefox latest version, I enabled cross-site cookies and followed all the FAQ/Guides and in general I tried just about everything but for some reason, when I'm watching just one stream through holodex, I cannot type in chat (I get the mouseover "login to your account or else!") prompt.

But during big collabs, it works fine. Here's a screenshot for proof:

![image](https://user-images.githubusercontent.com/8679333/139587528-76e1402f-f3c4-48af-b37d-b53eb44160e8.png)

I'm using hyperchat but I tried with/without, there is no change.

Any idea why this is happening? I can only think that it's somehow related to the fact that my youtube account is kind of a "brand" account (the one which I'm member of the channels with).

But that doesn't explain why it works for big collabs but doesn't work for single streams...

Anything else I can provide, let me know please...

**Comment by RiceCakess** [comment]
Not too sure why it's like that. Try disabling all your extensions. 

For accounts, try switching between branded/nonbranded accounts on youtube.com and then checking if it changes anything on Holodex.

**Comment by LittleVulpix** [comment]
Thanks. Actually the multiview was a red herring - because I set up the layout before the stream and then I closed/opened firefox at some point (and I have it set to reopend all previously opened tabs). On reopen, it worked. Even with a single-stream layout. So I experimented with a few addons and found out that it was caused by PrivacyBadger. Disabling the relevant cookie blocking inside the addon made it work fine without needing to restart the browser. I had forgotten I was using the addon since it works kinda autonomously, Can't believe I haven't tried it myself.

Thanks for the correct pointer.
